src/mutation.rs

```diff
@@ -0,0 +1,64 @@
+use async_graphql::{Context, Error, Object, Result, Upload};
+use bson::Uuid;
+use s3::Bucket;
+use serde::Serialize;
+use tokio::fs::File;
+use tokio::io::AsyncReadExt;
+
+/// Describes GraphQL media mutations.
+pub struct Mutation;
+
+#[Object]
+impl Mutation {
+    /// Uploads a media to MinIO.
+    ///
+    /// Saves the file under a new UUID with its suitable file extension in MinIO.
+    /// Sends an event when a media file was uploaded.
+    async fn upload_media<'a>(
```

Comment:
maybe I'm blind, but where is the auth check?
only employees/admins should be able to upload media files